### PR TITLE
Update repositories.txt

### PR DESCRIPTION
**Pull request by BenCestMoiQuoi**
Remove personal librairies

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @BenCestMoiQuoi. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

**Review by MatteoPologruto — APPROVED** [review]
Done:
```
2024/09/12 15:55:27 Loaded 7297 libraries from DB
Removing INSARIANNE
Success!
```

**Pull request merged**
